packages/trader/src/sass/app/_common/components/contract-replay.scss

```diff
@@ -0,0 +1,22 @@
+.vertical-tab__action-bar {
```

Comment:
Why is this leftover/override in Trader?

Reply:
This can be removed.